src/ui-test/lsp_extension_test.ts

```diff
@@ -38,6 +38,7 @@ describe('Language Support for Apache Camel extension', function () {
 		});
 
 		after(async function () {
+			this.timeout(10000);
 			await marketplace.close();
```

Comment:
Why do we need to add a timeout here? Why is it related to the node version change?